### ISSUE TITLE
make the api responsive during initialization

### ISSUE THREAD

**Issue by tarekziade** [loads/loads-broker#54]
populating the AMI is blocking startup, and the http API is not initialized during that phase.

**Comment by rpappalax** [comment]
@tarekziade by populating do you mean doing configuration of the AMI?

**Comment by tarekziade** [comment]
yeah, there's some blocking code that's pulling info from AWS on startup, to know which AMIs we should pull for CentOS.  

We can get rid of it with https://github.com/loads/loads-broker/issues/55 but there's no reason why the HTTP API are not initialized prior to this work.
